src/Graphics/OpenGLContext/GLFunctions.cpp

```diff
@@ -280,4 +280,10 @@ void initGLFunctions()
 	GL_GET_PROC_ADR(PFNGLCREATEBUFFERSPROC, glCreateBuffers);
 	GL_GET_PROC_ADR(PFNGLCREATEFRAMEBUFFERSPROC, glCreateFramebuffers);
 	GL_GET_PROC_ADR(PFNGLNAMEDFRAMEBUFFERTEXTUREPROC, glNamedFramebufferTexture);
+#ifdef EGL
+	if (g_glBufferStorage == nullptr)
+		g_glBufferStorage = (PFNGLBUFFERSTORAGEPROC) eglGetProcAddress("glBufferStorageEXT");
+	if (g_glTexStorage2D == nullptr)
+		g_glTexStorage2D = (PFNGLTEXSTORAGE2DPROC) eglGetProcAddress("glTexStorage2DEXT");
+#endif
```

Comment:
I know this is a little ugly, but with OpenGL ES they append "EXT" onto functions that come from an extension, so this is the best way I could think of to support ARB_buffer_storage and EXT_buffer_storage.